app/src/main/java/com/hercan/weatherapp2/presentation/model/CityWeatherItem.kt

```diff
@@ -0,0 +1,21 @@
+package com.hercan.weatherapp2.presentation.model
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class CityWeatherItem(
+    val city: String,
+    val temperature: Int,
+    val temperatureRange: String,
+    val weather: String
+) : Parcelable
+
+fun CityWeatherItem.getTemperatureRange(): String {
+    val temperatureRange = this.temperatureRange.split(" - ")
+    return temperatureRange.first().toString() + "° - " + temperatureRange.last().toString() + "°"
```

Comment:
daha okunakli olmasi acisindan soyle kullanabilirsin
```suggestion
    return "${temperatureMin}° - ${temperatureMax}°"
```
bunun dogrusu aslinda string.xml'de parametre alabilen bir string olusturup viewholder tarafinda o string degerine max min degerlerini vermek. 

https://medium.com/@sourav.bh/best-practices-for-using-string-resources-in-android-c947fdd1606a